content/webapp/components/EventsByMonth/group-event-utils.ts

```diff
@@ -84,11 +84,11 @@ function maxDate(dates: Date[]): Date {
 }
 
 function getEarliestStartTime({ times }: HasTimeRanges): Date {
-  return minDate(times.map(t => t.range.startDateTime));
+  return minDate(times.map(t => t.range.startDateTime).filter(isFuture));
 }
 
 function getLatestStartTime({ times }: HasTimeRanges): Date {
-  return maxDate(times.map(t => t.range.startDateTime));
+  return maxDate(times.map(t => t.range.startDateTime).filter(isFuture));
```

Comment:
These two functions are used in `groupEventsByMonth` to get the boundary of the group; as Vanesha correctly guessed, the October instance of the HIV/AIDs event is anchoring the "What's on" page in the past.